src/components/visual-query-builder/VisualEditor.tsx

```diff
@@ -50,6 +57,21 @@ export const VisualEditor: React.FC<VisualEditorProps> = ({
             <Preview rawSql={query.rawSql} />
           </EditorRow>
         )}
+        <EditorRow>
+          <QueryOptionGroup
+            title="Options"
+            collapsedInfo={[`Storage API: ${query.enableStorageAPI ? 'enabled' : 'disabled'}`]}
+          >
+            <InlineSwitch
+              id={`${htmlId}-storage-api`}
```

Comment:
Should we add a short info box that describes what turning "storage api" on will do? Perhaps also naming the switch label "Enable Storage API" or "Use Storage API" can help make this clearer.

Reply:
I renamed the field to Use Storage API everywhere.